4/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.8"
+services:
+  primarysilo:
+    build:  
+      context: .
+      dockerfile: ./ops/SiloHost/Dockerfile
+    environment:
+    - GATEWAYPORT=3000
+    - SILOPORT=2000
+    ports:
+      - "3000:8080"
+  secondarysilo:
+    build:  
+      context: .
+      dockerfile: ./ops/SiloHost/Dockerfile
+    environment:
+    - GATEWAYPORT=3000
+    - SILOPORT=2001
+    - PRIMARYSILOPORT=2000
+    - PRIMARYSILOADDRESS=primarysilo
+  siloclient:
+    build:  
+      context: .
+      dockerfile: ./ops/Client/Dockerfile
+    environment:
+    - GATEWAYPORT=3000
+    - SILODOMAIN=primarysilo
+  siloclient2:
+    build:  
+      context: .
+      dockerfile: ./ops/Client/Dockerfile
+    environment:
+    - GATEWAYPORT=3000
+    - SILODOMAIN=secondarysilo
```

Comment:
So the trick is to always address one silo and let Orleans do the routing. Technically, I'd probably advise we steer away from the `primary` silo wording as there isn't really a leader/first/primary silo. They should all be expendable peers.

Reply:
Finally, it seems that the dashboard port is overlooked (`http://localhost:8080/`) and one cannot see/monitor the cluster. If it was available, we would have seen two separate silos not forming a cluster due to slightly incorrect configuration. There's a good number of comments for this PR, let's maybe focus on getting the dashboard operational first.